test.md

```diff
@@ -23,6 +23,7 @@ This will allow `trap` to "bubble up" (more correctly, to "consume the continuat
     syntax Instr ::= Assertion
  // --------------------------
     rule <k> trap ~> (L:Label => .) ... </k>
+    rule <k> trap ~> (F:Frame => .) ... </k>
     rule <k> trap ~> (.Instrs => .) ... </k>
     rule <k> trap ~> (I:Instr => .) ... </k> requires notBool isAssertion(I)
```

Comment:
As a side note relating to #95, perhaps all these belong in `wasm.md`? If we make `Assertion` a subsort of `Bare`, then we won't need the final rule's side condition: a `trap` simply eats the instructions, frames, labels, definitions, declarations, etc, on the stack. But it doesn't eat `Bare`, or `Auxil`.

Reply:
I agree this should be in wasm.md